src/System.Linq.Expressions/tests/Invoke/InvocationTests.cs

```diff
@@ -36,19 +36,21 @@ public T Property
             }
         }
 
-        [Fact] // [Issue(3224, "https://github.com/dotnet/corefx/issues/3224")]
-        public static void SelfApplication()
+        // [Issue(3224, "https://github.com/dotnet/corefx/issues/3224")]
```

Comment:
Probably not. I just figured someone wanted the comment in there so I'd leave it.